styles/config/vocabularies/Canonical/accept.txt

```diff
@@ -44,11 +56,15 @@ VMware
 ZooKeeper
 config
 downtime
+eksctl
 keystore
+(?i)namespace
```

Comment:
That's an interesting case. This is not a proper name, so it is not always capitalized. But the particular false positive error is about `Namespace` used at the beginning of a sentence. So I added the case-insensitive syntax and now I wonder whether we need to add it to every non-capitalised term in this list (just in case they are used as the first word of a sentence).

Reply:
Good point(s). I like it.

Reply:
I spun this off into #80 so we can focus on the Spark related things in this PR :)